TrackletGraph.py

```diff
@@ -274,9 +294,12 @@ def get_proc_dict_from_config(fname_pconfig):
         for i,line_proc in enumerate(file_proc):
             # Processing module type
             proc_type = line_proc.split(':')[0].strip()
-            # temperary hack: DiskMatchCalculator-->MatchCalculator
+            # temporary hack: DiskMatchCalculator-->MatchCalculator
             if proc_type == "DiskMatchCalculator":
                 proc_type = "MatchCalculator"
+            # temporary hack: FitTrack-->TrackBuilder
```

Comment:
Could @aryd please modify the emulation, to avoid this hack, by renaming "FitTrack" --> "TrackBuilder" in wires.dat?